api/reference/virtual_templates.adoc

```diff
@@ -0,0 +1,163 @@
+[[virtual-template-requests]]
+== Virtual Template Management
+
+Management of Virtual Templates is provided via the following collection:
+
+[source, data]
+----
+/api/virtual_templates
+----
+
+Full CRUD actions on virtual templates are available:
+
+* link:#querying-virtual-templates[Querying Virtual Templates]
+* link:#creating-virtual-templates[Creating Virtual Templates]
+* link:#editing-virtual-templates[Editing Virtual Templates]
+* link:#deleting-virtual-templates[Deleting Virtual Templates]
+
+[[querying-virtual-templates]]
+=== Querying Virtual Templates
+
+Querying all virtual templates in the system is simply:
+
```

Comment:
remove "simply". you don't need it. Either use "Getting" or "Querying" to be consistent.